GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloHGABlockV.cfg

```diff
@@ -5,37 +5,45 @@
 	@manufacturer = Boeing
 	@description = A collapsible relay antenna consisting of a high gain dish, for transmitting the many exciting things you've discovered. Intended to be placed on the mounting bracket of the Apollo Service Engine Block V. FICTIONAL (Eyes Turned Skyward)
 
+	%RSSROConfig = True
+
 	@MODEL
 	{
 		@model = ROCapsules/Assets/BDB/bluedog_Apollo_Block5_HGA
-		%rotation = 0.0, -60.0, 0.0
+		%rotation = 0.0, 45.0, 0.0
 	}
-	%RSSROConfig = True
+
 	@rescaleFactor = 1.5646
 	@mass = 0.025
 
 	%TechRequired = commandModules
 	
-	%tags = aerial antenna radio signal transmit comm communicat dish Apollo Kane block ?5 ?v
+	%tags = apollo, block-V, block-5, eyes turned skyward
+
+	@MODULE[ModuleAnimateGeneric]
+	{
+		@animationName = deploy
+	}
 
 	@MODULE[ModuleRTAntenna]:NEEDS[RemoteTech]
 	{
 		@Mode1OmniRange *= 2
 	}
 }
 
+//	================================================================================
+//	Configure RealAntennas
+//	================================================================================
+// A steerable unified S-band high-gain antenna for long-range communications with Earth was mounted on the aft bulkhead.
+// https://en.wikipedia.org/wiki/Apollo_command_and_service_module#Communications_system
+
 @PART[ROC-ApolloHGABlockV]:BEFORE[RealAntennas]
 {
 	!MODULE[ModuleDataTransmitter],* {}
 	MODULE
 	{
 		name = ModuleRealAntenna
 		antennaDiameter = 1.5	//Guess, no documentation
-	}
-	MODULE
-	{
-		name = ModuleRealAntenna
-		referenceGain = 3.0		//Guess based on Block II
-		RFBand = S
```

Comment:
The ModuleRealAntenna modules were intentionally duplicated to provide both an omni and a steerable dish. Is that not correct?